src/views/CampusSafety/views/LostAndFound/views/MissingItemCreate/index.tsx

```diff
@@ -65,21 +80,25 @@ const MissingItemFormCreate = () => {
     return Object.keys(errors).length === 0;
   };
 
-  // Handle form data changes
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const { name, value, type, checked } = e.target;
 
     if (name === 'stolen') {
-      setStolenModalOpen(checked); // Open modal if stolen checkbox is checked
-      setFormData((prevData) => ({
-        ...prevData,
-        [name]: checked,
-      }));
-    } else if (name === 'dateLost') {
```

Comment:
I believe it was because i changed how the handlechange was being handled and didnt need the part to format the date, since its already being done on the confirmation page.